wp-content/themes/csisnuclearnetwork/assets/_scss/pages/home.scss

```diff
@@ -493,4 +493,129 @@
       }
     }
   }
+
+  .newsletter-block-acf {
+    display: grid;
+    grid-template-areas: 
+      'header'
+      'daily'
+      'monthly';
+    padding-top: rem(56);
+    padding-bottom: rem(72);
+    @include full-width-background(linear-gradient(180deg, #153b6a 0%, #215f8c 100%));
+    background: linear-gradient(180deg, #153b6a 0%, #215f8c 100%);
+
+    @include breakpoint(medium) {
+      grid-template-areas: 
+        'header header'
+        'daily monthly';
+      column-gap: rem(40);
+    }
+
+    @include breakpoint(large) {
+      grid-template-areas: 'header daily monthly';
+      column-gap: rem(40);
+    }
+
+    h3 {
+      grid-area: header;
+      color: $color-white-100;
+      @extend %text-style-heading-x-large-strong;
+    }
+
+    p {
+      flex-basis: 100%;
+      margin-top: rem(8);
+      margin-bottom: rem(24);
+      @extend %text-style-paragraph-large;
+      @extend %text-style-short;
+    }
+
+    .daily,
+    .monthly {
+      display: flex;
+      flex-wrap: wrap;
+      align-content: start;
+      color: $color-white-190;
+      
+      @include breakpoint(large) {
+        max-width: 30vw;
+        padding-top: 0 !important;
+      }
+
+      /* stylelint-disable */
+      h3 {
+        @extend %text-style-heading-large-strong;
+        margin-top: 0;
+      }
+      /* stylelint-enable */
+    }
+
+    .daily {
+      grid-area: daily;
+      padding-top: rem(16);
+      padding-bottom: rem(32);
+    }
+
+    .monthly {
+      grid-area: monthly;
+      padding-top: rem(24);
+
+      @include breakpoint(medium) {
+        padding-top: rem(16);
+      }
+    }
+
+    .recent-news {
+      flex-basis: 100%;
+
+      @include breakpoint(small) {
+        flex-basis: 50%;
+        margin-left: rem(48);
+        padding-top: rem(8);
+      }
+
+      @include breakpoint(medium) {
+        flex-basis: 100%;
+        margin-left: 0;
+        padding-top: 0;
+      }
+      @include breakpoint(xlarge) {
+        flex-basis: 50%;
+        margin-left: rem(48);
+        padding-top: rem(8);
+      }
+
+/* stylelint-disable */
+      :nth-child(2) {
+        padding-top: rem(16);
+      }
+
+      a {
+        display: block;
```

Comment:
I would recommend making this `display: flex; align-items: center;` so the text and the icon vertically match up. Right now the icon is out of alignment with the text.